packages/sitecore-jss/src/utils/env-helper.ts

```diff
@@ -0,0 +1,28 @@
+export class EnvHelper {
+  /**
+   * Entry method to parse multi or single value env variables
+   * @param {string} envValue
+   * @returns Record, Array or string depending on input format
```

Comment:
It doesn't return Array anymore
```suggestion
   * @returns Record or string depending on input format
```